### PR TITLE
Cherry-pick federated traits test fix into develop

### LINKED ISSUE #2616: FederatedStore traits tests are inconsistent
Sometimes the FederatedStoreGetTraitsTest and FederatedGraphStorageTraitsTest will fail due to not being able to add graphs to the FederatedStore.
This is an inconsistent failure, commonly seen due to the FederatedStore cache not properly being cleared before each test.
@r32575 fixed this in https://github.com/gchq/Gaffer/pull/2583 with https://github.com/gchq/Gaffer/pull/2583/commits/0788bc4790ae431f1862b7611255ab42778ee3eb but the fix should also be cherry-picked over to develop.

### PR DESCRIPTION
**Pull request by t92549**
Cherry picked from https://github.com/gchq/Gaffer/pull/2583/commits/0788bc4790ae431f1862b7611255ab42778ee3eb

# Related Issue

- Resolve #2616

### CODE REVIEW

File: store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorageTraitsTest.java [hunk 1]
```diff
@@ -112,11 +115,14 @@
...
-    private static final StoreProperties ACCUMULO_PROPERTIES = StoreProperties.loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseAccumuloStore.properties"));
-    private static final StoreProperties MAP_PROPERTIES = StoreProperties.loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseMapStore.properties"));
+    private static final StoreProperties ACCUMULO_PROPERTIES = StoreProperties
+            .loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseAccumuloStore.properties"));
+    private static final StoreProperties MAP_PROPERTIES = StoreProperties
+            .loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseMapStore.properties"));
 
     @BeforeEach
     public void setUp() throws Exception {
+        CacheServiceLoader.shutdown();
```
Comment (t92549) [inline store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorageTraitsTest.java:125]: The diff looks large but effectively this is the fix

File: store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreGetTraitsTest.java [hunk 2]
```diff
@@ -112,16 +116,19 @@
...
+            .loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseAccumuloStore.properties"));
+    private static final StoreProperties MAP_PROPERTIES = StoreProperties
+            .loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseMapStore.properties"));
     private FederatedStore federatedStore;
 
-    @Before
-    public void setUp() throws Exception {
+    @BeforeEach
+    public void beforeEach() throws Exception {
+        CacheServiceLoader.shutdown();
```
Comment (t92549) [inline store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreGetTraitsTest.java:127]: As well as this

**Review by t92549 — COMMENTED** [review]
Highlighted the actual fix, rest of the changes are formatting ones done in https://github.com/gchq/Gaffer/pull/2583

**Review by GCHQDeveloper314 — APPROVED** [review]
I've run the tests using this branch and can confirm the failure no longer occurs and has been fixed.

**Pull request merged**